setup.cfg

```diff
@@ -1,5 +1,5 @@
 [flake8]
-extend-ignore = I900,Q000,WPS210,PD011
+extend-ignore = I900,Q000,WPS210,PD011,S104
```

Comment:
For DX, I'd rather setup 0.0.0.0 by default.

In the end, we should use env vars to config host and port anyway...